react/Layout/Layout.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styles from './Layout.styl'
+import cx from 'classnames'
+
+export const Layout = ({ children, className, monoColumn, ...rest }) => {
+  return (
+    <div
+      className={cx(
+        monoColumn ? styles['o-layout'] : styles['o-layout--2panes'],
```

Comment:
I think o-layout--2panes should not be with `--` as it is not a variant. I would think a variant should be applied additionally to the normal class. In this case, it is one or the other.

Reply:
true. It used to be a variant but I didn't change the naming. 👍